test/track.test.js

```diff
@@ -55,6 +55,50 @@ describe('ravelin.track', function() {
       r = new Ravelin(isolate({key: key, api: '/', page: {section: 'test'}}));
     });
 
+    it('can be manually initialised', function(done) {
+      var key = this.test.fullTitle();
+      var errored = false;
+      var waited = false;
+      xhook.before(function(req) {
+        if (!keysMatch(req, key)) return {status: 204};
+
+        if (!waited) {
+          errored = true;
+          done('received an API request but should have gotten none: ' + JSON.stringify(req));
+        } else {
+          r.core.ids().then(function(ids) {
+            var loadEvent = JSON.parse(req.body).events[0];
+            expect(loadEvent).to.have.property('eventType', 'track');
+            expect(loadEvent.libVer).to.match(expectedVersion);
+            expect(loadEvent.eventData).to.eql({eventName: 'PAGE_LOADED', properties: {section: 'test'}});
+            expect(loadEvent.eventMeta.trackingSource).to.be('browser');
+            expect(loadEvent.eventMeta.ravelinDeviceId).to.be(ids.device);
+            expect(loadEvent.eventMeta.ravelinSessionId).to.be(ids.session);
+          }).then(done, done);
+        }
+        return {status: 204};
+      });
+      setTimeout(function() {
+        if (errored) return;
+        waited = true;
+        r.track.init({section: 'test'});
+      }, 200);
+      r = new Ravelin(isolate({key: key, api: '/', track: false}));
+    });
+
+    it('is suppressed by page: false', function(done) {
```

Comment:
Very minor thing but this test name initially confused me, what about something like `is suppressed when page: false is set`

Reply:
Renamed to "doesnt send a page-loaded event when initialised with page:false"